js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -269,16 +284,17 @@ export default class TypedInput extends Component {
    * @see https://github.com/facebook/react/issues/1549
    */
   renderFloat (value = this.props.value, onChange = this.onChange) {
-    const { allowCopy, label, error, hint, min, max, readOnly } = this.props;
+    const { allowCopy, className, label, error, hint, min, max, readOnly } = this.props;
     const param = this.getParam();
 
     const realValue = value
-      ? (new BigNumber(value))[readOnly ? 'toFormat' : 'toNumber']()
+      ? this.getNumberValue(value)[readOnly ? 'toFormat' : 'toNumber']()
```

Comment:
Same breakage comment here since the function may return something without toFormat/toNumber